hack/libpfm.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -f /etc/redhat-release ]; then
+    # RedHat or CentOS 
+    sudo yum -y update && \
+        sudo yum -y install cmake wget && \
+        sudo yum -y groupinstall "Development Tools"
+elif [ -f /etc/debian_version ]; then
+    # Debian or Ubuntu 
+    sudo apt-get update && \
+        sudo apt-get -y install bash build-essential cmake wget
+elif [ $(uname) == "Darwin" ]; then
+    echo "skip macOS"
```

Comment:
@bowen-intel hi, here just echo "skip macOS" is not enough, still try to install the libpfm on macOS.

Reply:
Sorry, I misunderstand @zwzhang0107 comment. If "skip for macOS" is skip libpfm4 install, I will add `exit 0` for macOS.

Reply:
Sorry, I misunderstand @zwzhang0107 comment. If "skip for macOS" is for `libpfm4` installation, I will add `exit 0` for macOS.